Commands/Commands/uservoice.js

```diff
@@ -1,10 +1,10 @@
-var commands = []
+let commands = []
 
-// var checker = require('../../Utils/access_checker')
-var logger = require('../../Utils/error_loggers')
-var config = require('../../config.js')
-var Entities = require('html-entities').AllHtmlEntities
-var UVRegex = /https?:\/\/[\w.]+\/forums\/(\d{6,})-[\w-]+\/suggestions\/(\d{7,})(?:-[\w-]*)?/
+// const checker = require('../../Utils/access_checker')
```

Comment:
Constants aren't program wide, they're basically variables, except you can't reassign or redeclare them.